frontend/src/Artist/Details/ArtistDetailsSeason.js

```diff
@@ -51,12 +51,18 @@ class ArtistDetailsSeason extends Component {
     const {
       name,
       onExpandPress,
-      items
+      items,
+      uiSettings
     } = this.props;
 
     const expand = _.some(items, (item) => {
-      return isAfter(item.releaseDate) ||
-             isAfter(item.releaseDate, { days: -365 });
+      return ((item.albumType === 'Album') && uiSettings.expandAlbumByDefault) ||
+        ((item.albumType === 'Single') && uiSettings.expandSingleByDefault) ||
+        ((item.albumType === 'EP') && uiSettings.expandEPByDefault) ||
+        ((item.albumType === 'Broadcast') && uiSettings.expandBroadcastByDefault) ||
+        ((item.albumType === 'Other') && uiSettings.expandOtherByDefault) ||
+        isAfter(item.releaseDate) ||
+        isAfter(item.releaseDate, { days: -365 });
```

Comment:
Should we remove this, if not then items will expand even if user has setting set not too.